### ISSUE TITLE
remove calls to get `getOutboundLink` leftover from Google Analytics

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#718]
## Type of Change
- [ ] New Feature Request
- [x] Documentation / Website
- [ ] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
Noticed while clicking on an (external) link, this console error was showing
![Screen Shot 2021-09-07 at 4 58 37 PM](https://user-images.githubusercontent.com/895923/132410020-62bbe0e1-22dd-4744-81f2-32b58dce333c.png)

## Details
As part of the migration away from #496 , I forgot to take these calls out 🤦‍♂️ 

Basically just need to remove all usages of [`getOutboundLink`](https://github.com/ProjectEvergreen/greenwood/blob/master/www/components/header/header.js#L53) from the _www/_ pages.
